spec/models/comment_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
You need to add unit tests for the comment model

Reply:
All fixed!